README.md

```diff
@@ -2,41 +2,38 @@
 
 Welcome! This repository contains documentation and resources for [Cilium](https://github.com/cilium/cilium) Installfest.
 
-The Installfest is a community event where experienced Cilium users will help you set up Cilium and give you a hands-on overview of its capabilities.
+The Installfest is a live community event where experienced Cilium users will help you set up Cilium and give you a hands-on overview of its capabilities.
 Our goal is to provide anyone interested in Cilium with step-by-step instructions for quickly installing Cilium and manipulating it while illustrating real-world use cases.
 
-> Note: while the Installfest is conceived as a live event where experienced users can help you get up to speed, the step-by-step instructions below can still be followed offline should you prefer to run through on your own.
+> Note: the step-by-step instructions below can still be followed offline should you prefer to run through on your own.
 
 ## Before the Installfest
 
 To ensure a smooth Installfest, please make sure your local environment is set up and ready before the event.
 
 ### Technical prerequisites
 
-- Familiarity with Kubernetes
+- Familiarity with command line usage and basic Kubernetes operations
 - A Linux or Intel-based Mac machine with `kubectl` installed: https://kubernetes.io/docs/tasks/tools/#kubectl
-- Minikube (>= 1.5.2): https://minikube.sigs.k8s.io/docs/start/
+- Minikube installed (>= 1.5.2): https://minikube.sigs.k8s.io/docs/start/
 
-> Note: while we strongly recommend using Minikube, the Installfest may also be followed on any GKE, EKS, or AKS cluster, though you will not get support from the Installfest staff in case of cluster-related issues during the event.
+> ⚠️ Minikube's `docker` driver does not support `btrfs` at the moment (see [kubernetes/minikube#7923](https://github.com/kubernetes/minikube/issues/7923)). Be wary of this if you are on Fedora / openSUSE (which use `btrfs` by default). Potential workaround in [kubernetes/minikube#2990](https://github.com/kubernetes/minikube/pull/12990), or use another driver.
```

Comment:
Yea, my intent with this was to mention that `btrfs` is not supported, and since Fedora / openSUSE have it by default of course the issue will happen. I expect if someone using another distro has configured `btrfs`, they would know they did so and understand this also applies to them. Open to suggestions regarding how to word that better :D

Reply:
You know what, let me retract that, it's fine, surely it's enough :+1: